### ISSUE TITLE
Javascript heap out of memory

### ISSUE THREAD

**Issue by erwanp** [suzil/radis-ui#113]
Hello @suzil ,

I tried to run the server here. Backend started properly 

```
python backend.py
 * Serving Flask app "backend" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
```

but frontend fails : 

```
$ yarn start
yarn run v1.22.10
$ react-scripts start
i ｢wds｣: Project is running at http://192.168.71.1/
i ｢wds｣: webpack output is served from
i ｢wds｣: Content not from webpack is served from D:\GitHub\radis-ui\frontend\public
i ｢wds｣: 404s will fallback to /
Starting the development server...

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 00007FF612CC5EBF napi_wrap+114095
 2: 00007FF612C70B46 v8::base::CPU::has_sse+66998
 3: 00007FF612C71946 v8::base::CPU::has_sse+70582
 4: 00007FF613486E4E v8::Isolate::ReportExternalAllocationLimitReached+94
 5: 00007FF61346EF21 v8::SharedArrayBuffer::Externalize+833
 6: 00007FF61333B18C v8::internal::Heap::EphemeronKeyWriteBarrierFromCode+1436
 7: 00007FF6133463C0 v8::internal::Heap::ProtectUnprotectedMemoryChunks+1312
 8: 00007FF613342EE4 v8::internal::Heap::PageFlagsAreConsistent+3204
 9: 00007FF6133386E3 v8::internal::Heap::CollectGarbage+1283
10: 00007FF613336D54 v8::internal::Heap::AddRetainedMap+2452
11: 00007FF61335809D v8::internal::Factory::NewFillerObject+61
12: 00007FF6130BE06F v8::internal::interpreter::JumpTableTargetOffsets::iterator::operator=+1295
13: 00007FF6138DB3BD v8::internal::SetupIsolateDelegate::SetupHeap+546637
14: 000000C904570EB1
error Command failed with exit code 134.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

I don't know if it helps at this point !

**Comment by suzil** [comment]
Which version of node are you running? `$ node --version`

I found this issue and it seems some people still have the problem you've encountered: https://github.com/facebook/create-react-app/issues/8096 So maybe there are some things to try there.

I've tried starting the app with node 10, 12, 14 and all versions I don't have any issues. Will try to look into it some more.

**Comment by erwanp** [comment]
```
 node --version
v12.18.3
```
on Windows

**Comment by erwanp** [comment]
They mention 
> Node 15 didn't fix it for me, but --optimize-for-size did! You can't include it in NODE_OPTIONS though, you have to insert it into your script(s) in your package.json

**Comment by erwanp** [comment]
I fixed that by updating to node v15.5.0.  

(and then got the #148 error, fixed too) 

We can close.
